app/src/test/java/gov/va/vro/EvidenceSummaryDocumentProcessorTest.java

```diff
@@ -33,6 +33,9 @@
 public class EvidenceSummaryDocumentProcessorTest extends BaseControllerTest {
 
   @Value("classpath:test-data/pdf-generator-mas.json")
+  private Resource pdfGeneratorMas;
+
+  @Value("classpath:test-data/pdf-generator-input-01.json")
```

Comment:
is 01 for v1? i'd maybe name it v1 if you could to be clearer whats going on. (if not, hopefully i'll figure it out as i go).